quickjs-libc.c

```diff
@@ -3954,6 +3967,36 @@ void js_std_add_helpers(JSContext *ctx, int argc, char **argv)
     JS_FreeValue(ctx, global_obj);
 }
 
+static void js_std_rt_finalizer(JSRuntime *rt, void *arg)
+{
+    JSThreadState *ts = arg;
+    struct list_head *el, *el1;
+
+    list_for_each_safe(el, el1, &ts->os_rw_handlers) {
+        JSOSRWHandler *rh = list_entry(el, JSOSRWHandler, link);
+        free_rw_handler(rt, rh);
```

Comment:
On third thought... this isn't quite right either because we still need to be able to call JS_FreeValueRT during cleanup.

I'm going to keep js_std_free_handlers and only move freeing JSThreadState here.